src/views/forms/ClosePositionForm.tsx

```diff
@@ -381,6 +398,13 @@ const $ToggleGroup = styled(ToggleGroup)`
   }
 `;
 
+const $AlertContent = styled.div`
+  display: inline-block;
+  a {
+    margin-left: 0.5ch;
+  }
```

Comment:
if the `a` is inlined in the `<span>` I think we can just change `display: inline-block` so it respects spaces

Reply:
ooh good point - will follow up in https://github.com/dydxprotocol/v4-web/pull/963#discussion_r1735277963